packages/scss/src/components/_toasts.scss

```diff
@@ -0,0 +1,62 @@
+.toasts {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: wrap;
+	font-size: _theme("sizes.small.font-size");
+	line-height: _theme("sizes.small.line-height");
+	position: fixed;
+	right: _component("toasts.right");
+	top: _component("toasts.top");
+	width: _component("toasts.width");
+	z-index: 9999;
+}
+
+.toasts-item {
+	animation: toast _theme("commons.animations.durations.standard") 1;
+	background: rgba(0, 0, 0, .85);
+	border-radius: _theme("commons.border.radius");
+	color: white;
+	margin-bottom: _component("toasts.margin-bottom");
+	margin-left: auto;
+	overflow: hidden;
+	padding: _component("toasts.padding");
+	position: relative;
+	transform-origin: top;
+}
+
+.toasts-item-kill {
+	background: transparent;
+	border: 0;
+	bottom: 0;
+	color: white;
+	cursor: pointer;
+	height: 1.25rem;
+	margin: auto;
+	padding: 0;
+	position: absolute;
+	right: .5rem;
+	top: 0;
+	transition: opacity _theme("commons.animations.durations.fast") ease;
+	width: 1.25rem;
+
+	&:hover {
+		opacity: .66;
+	}
+
+	&::after {
+		@include makeIcon("cross");
+		font-size: _theme("sizes.standard.font-size");
+	}
+}
+
+@keyframes toast {
```

Comment:
You might want to use the keyframes in utilities/animations files.
Either way, if you prefer using one here, use translations instead of changing top for better performances.

Reply:
Indeed, keyframes was previously used for advanced animations (slide in / out) ans it was necessary to use a top positionning to affect others toasts positions. 
But as the toasts functionning changed, it's not needed anymore.